apps/etterlatte-grunnlag/src/main/resources/db/migration/V27__sette_innsender_pesys_til_null.sql

```diff
@@ -0,0 +1,5 @@
+update grunnlagshendelse
+set opplysning = jsonb_set(opplysning::jsonb, '{innsender}', 'null')
+where LENGTH(opplysning::JSON ->> 'innsender') < 11 AND opplysning_type = 'PERSONGALLERI_V1';
+
+-- todo hvordan gjøre denne med rollback?
```

Comment:
kan man feks kopiere hele grunnlagshendelse-tabellen før man gjør endringen i eksisterende? Det gir en grei mulighet for rollback dersom man umiddelbart verifiserer.

```CREATE TABLE grunnlagshendelse_backup AS TABLE grunnlagshendelse;```

Reply:
Ja liker ideen der! Det er absolutt en mulighet.

Men er det noe annet smart man kan sjekke på her etter kjøringen som kaster sql exception hvis innholdet blir oppdatert feil?

Reply:
Kunne det feks vært å traversere json i et utvalg av de som oppdateres i etterkant?
For hvis jsonen blir borte eller satt til null vil det kræsje og reversere updaten